modules/n1ql/pages/n1ql-language-reference/selectintro.adoc

```diff
@@ -380,7 +380,7 @@ See xref:n1ql-language-reference/subqueries.adoc[Subqueries] for more details.
 [source,N1QL]
 ----
 SELECT *
-FROM   (SELECT t.airportname
+FROM   (SELECT s1.airportname
```

Comment:
Reject this change. In the original example, the inmost subquery selects a single airport in the United States and gives it the alias `t`. The next subquery selects the airport name from `t` and gives it the alias `s1`. The outermost query selects everything from `s1` and gives it the alias `s2`. If we make the change given in this line of the pull request, the query returns an empty object, and the results in lines 391-401 are incorrect.

Revert this line to the original as follows:

```suggestion
FROM   (SELECT t.airportname
```